projects/application/package-lock.json

```diff
@@ -14483,87 +14483,6 @@
         "chart.js": "^2.9.2",
         "ng2-charts": "^2.3.0",
         "tslib": "^1.9.0"
-      },
-      "dependencies": {
```

Comment:
Why is this removed?

Reply:
Not removed, only moved as far as I can see. package-lock.json is auto-generated from package.json.